marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx

```diff
@@ -15,7 +15,7 @@ describe('Create/Edit Mapping Step artifact component', () => {
     expect(getByPlaceholderText('Enter description')).toBeInTheDocument();
     expect(getByLabelText('Collection')).toBeInTheDocument();
     expect(getByLabelText('Query')).toBeInTheDocument();
-    expect(getByPlaceholderText('Please select')).toBeInTheDocument();
+    expect(getByPlaceholderText('Enter Collection')).toBeInTheDocument();
```

Comment:
I believe there's another JIRA for this where the decision was to do "Enter collection name". "Enter collection" is fine, but "C" needs to be "c".

Reply:
@rjrudin I noticed there were some discrepancies with the casing on the relevant forms. There's 'Enter name' and 'Enter description' but for the Query input that is next to Collection, it uses 'Enter Source Query' so I chose to keep the casing consistent with that. Agree that 'Enter collection name' may be better. Maybe a question for UX? @jbelonoj

Reply:
Thanks for bringing this up, @briantang! Yes, for ALL placeholder text we agreed on this format: "Enter collection name". Also, please note that we do not start with "Please", we start with "Enter" or "Select" in most cases. First word starts with capital letter, all following words start with lowercase letters.

Reply:
Ok @jbelonoj thanks for clearing that up. Will also fix the casing in Query from 'Enter Source Query' to 'Enter source query' as part of this PR.